tests/test_server.py

```diff
@@ -2090,6 +2090,8 @@ def mock_os_path_exist(file_name):
             "The first missing WAL file is "
             "000000010000000000000003"
         )
+        _out, err = capsys.readouterr()
```

Comment:
```suggestion
        _, err = capsys.readouterr()
```

We can just assign to `_` so PyLance stops complaining in the IDE 😄